.envrc

```diff
@@ -32,7 +34,7 @@ Follow them, and then you'll need to redo direnv by running "direnv allow".${res
 
 If you're stuck or have questions, ask in #discuss-dev-tooling.
 EOF
-    report_to_sentry
+    [ -z "${SENTRY_DEVENV_NO_REPORT+x}" ] && report_to_sentry
```

Comment:
We were jumping inside of `report_to_sentry` and installing `sentry-cli` even if we were not going to report anything due to the lack of `SENTRY_DSN`.